mariza/styles.css

```
@@ -0,0 +1,176 @@
+* {
+  box-sizing: border-box;
+}
+.header {
+  background-color: #ff8c00;
+  display: inline-block;
+  color: #ffffff;
+  font-family: "Open Sans", sans-serif;
+  width: 100%;
+}
+h1 {
+  background-color: #ff8c00;
+  font-size: 30px;
+  font-style: regular;
+  font-weight: medium;
+  font-family: "Raleway", sans-serif;
+  color: white;
+}
+.hours {
+  display: inline-block;
+  background-color: #ff8c00;
+  font-size: 12px;
+  color: white;
+  font-weight: light;
```

See previous note about `font-weight`
```suggestion
  font-weight: 300;
```